gui-tk/wiring.tcl

```diff
@@ -1191,6 +1198,7 @@ proc setIntegralIValue {} {
     global constInput
     set item minsky.canvas.item
     $item.description $constInput(Name)
+    $item.name $constInput(Name)
```

Comment:
My understanding is that at this point, item is an integral, so this change would be wrong.

Reply:
Oops, I forgot that there. Thanks, I've deleted it.